components/camel-jt400/src/main/java/org/apache/camel/component/jt400/Jt400Endpoint.java

```diff
@@ -36,14 +36,20 @@
 import org.apache.camel.util.URISupport;
 
 /**
- * Exchanges messages with an AS/400 system using data queues or program call.
+ * Exchanges messages with an AS/400 system using data queues, message queues, or program call.
  */
 @UriEndpoint(firstVersion = "1.5.0", scheme = "jt400", title = "JT400", syntax = "jt400:userID:password/systemName/objectPath.type", category = {Category.MESSAGING})
 public class Jt400Endpoint extends ScheduledPollEndpoint implements MultipleConsumersSupport {
 
     public static final String KEY = "KEY";
     public static final String SENDER_INFORMATION = "SENDER_INFORMATION";
 
+    // Used only for message queue support
+    public static final String MESSAGE_ID = "jt400.MESSAGE_ID";
```

Comment:
These are the Camel headers isn't? I think it would make sense to have the headers with `Camel` prefix in order to standardize with other components as well, e.g: `CamelJt400MessageID`.
Also, it would make sense to place the constants into a singleton class to ease the usability for users.
 Example from [recent component](https://github.com/apache/camel/blob/master/components/camel-azure-storage-queue/src/main/java/org/apache/camel/component/azure/storage/queue/QueueConstants.java).

Reply:
I used the "jt400." prefix to mimic [the Kafka component](https://camel.apache.org/components/latest/kafka-component.html). But I see even that component's new addition has "CamelXxxxXxxx" format ("CamelKafkaManualCommit"). I will change accordingly. Thanks!

Reply:
Resolved at https://github.com/apache/camel/pull/3917/commits/4e6ee7fc5fff24fc68257a7f54c5ad570401c35f